exercises/codec-server/solution/codec-server/main.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"strconv"
+
+	edu_converters_go_code "github.com/temporalio/edu-converters-go-code/exercises/codec-server/solution"
+
+	"go.temporal.io/sdk/converter"
+	"go.temporal.io/server/common/log"
+	"go.temporal.io/server/common/log/tag"
```

Comment:
IMO we should not depend on the `go.temporal.io/server` repo/package in any of our code (especially if you don't need the OIDC stuff). That brings in our entire server as a dependency. https://github.com/temporalio/samples-go/tree/main/codec-server/codec-server should be updated too. Note https://github.com/temporalio/samples-go/tree/main/encryption/codec-server has no such dependency.

Reply:
I'll see what I can do to pull this out, thanks!